fiftyone/core/collections.py

```diff
@@ -3238,6 +3238,51 @@ def exclude_frames(self, frame_ids, omit_empty=True):
             fos.ExcludeFrames(frame_ids, omit_empty=omit_empty)
         )
 
+    @view_stage
+    def exclude_groups(self, group_ids):
+        """Excludes the groups with the given IDs from the grouped collection.
+
+        Examples::
+
+            import fiftyone as fo
+            import fiftyone.zoo as foz
+
+            dataset = foz.load_zoo_dataset("quickstart-groups")
+
+            #
+            # Select some specific groups by ID
```

Comment:
nit: update docstrings to say "exclude" rather than "select" in a few places here

Reply:
fixed